test/jdk/javax/swing/JTableHeader/6889007/bug6889007.java

```diff
@@ -69,6 +69,7 @@ public static void main(String[] args) throws Exception {
                 frame.add(th);
                 frame.pack();
                 frame.setLocationRelativeTo(null);
+                frame.setAlwaysOnTop(true);
```

Comment:
Why do we need to keep the window on top?

Reply:
In windows primarily, I see sometimes the frame is rendered below some if there are other windows present ie remains hidden and mouse cursor is moved which looks odd, so I ensured the frame is always on top for the test